### PR TITLE
Improve README setup

### LINKED ISSUE #376: Improve README installation to be clearer

### PR DESCRIPTION
**Pull request by ckcollab**
# @ mention of reviewers
@Germs31 

# A brief description of the purpose of the changes contained in this PR.
Make setup a bit clearer

# Issues this PR resolves
#376 

# TODO:
- [ ] Test locally with no `DEBUG`, serves OK?

# A checklist for hand testing
- [ ] these instructions work :)

# Checklist
- [x] Code review by me 
- [x] Hand tested by me 
- [x] I'm proud of my work
- [ ] Code review by reviewer
- [ ] Hand tested by reviewer
- [ ] Ready to merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,14 @@
 ```
 $ cp .env_sample .env
 $ docker-compose up -d
+$ docker-compose exec django ./manage.py migrate
+$ docker-compose exec django ./manage.py generate_data
+$ docker-compose exec django ./manage.py collectstatic --noinput
```
Comment (Germs31) [inline README.md]: thanks for adding this

**Comment by ckcollab** [comment]
Waiting on merging this until we figure out some steps to maybe speed things up (running extremely slowly on German's machine at the moment)

**Comment by jimmykodes** [comment]
One slight speed boost you can get is from using a docker-compose.override.yml to turn off builder and run npm-watch in the terminal rather than a docker image. May help at least a little?

**Comment by ckcollab** [comment]
@jimmykodes ty!!

**Pull request merged**